scripts/stablex/setup_thegraph_data.js

```diff
@@ -0,0 +1,233 @@
+const { solutionSubmissionParams, generateTestCase } = require("../../test/resources/examples")
+
+const { makeDeposits, placeOrders } = require("../../test/stablex/stablex_utils")
+const { closeAuction } = require("../../scripts/stablex/utilities.js")
+const { decodeOrdersBN } = require("../../src/encoding")
+
+const { toETH, feeAdded, feeSubtracted, ERROR_EPSILON } = require("../../test/resources/math")
+const { waitForNSeconds } = require("../../test/utilities")
+
+const BatchExchange = artifacts.require("BatchExchange")
+const ERC20Mintable = artifacts.require("ERC20Mintable.sol")
+
+async function _getOwl() {
```

Comment:
It would be good if you could put the helper functions into the utiltities.js file. There they are much easier to find and we can also use them in unit tests, hence refactoring errors are caught better.

Much of the OWL minting and token adding functionality could/should also be used from here:
https://github.com/gnosis/dex-contracts/blob/master/scripts/stablex/utilities.js#L38

Reply:
Okay, I'll move there more logic and refactor a bit